### DISCUSSION TITLE
Make forgified fabric api compatible with older forge version

### DISCUSSION THREAD

**Discussion by yousifhub** [Sinytra/ForgifiedFabricAPI#138] [Ideas]
im playing a modpack with my friend and we wanted to add mods but some good mods are a fabric mod so i found this mod and seems its not compatible with 1.19.2 so all im asking make the mod compatible with older version starting from 1.16

**Comment by yousifhub** [comment]
one thing i also want is to make is fabric being able to run forge mods as well maybe call it "Fabricated Forge Mod Loader"

**Comment by Su5eD** [comment]
Backports are not planned, please see our FAQ for more information. https://sinytra.org/docs/connector/faq

**Reply by yousifhub** [reply]
i see
